macosx/CreatorWindowController.mm

```diff
@@ -199,8 +199,8 @@ - (void)awakeFromNib
         URLByAppendingPathComponent:[name stringByAppendingPathExtension:@"torrent"]];
     if (!self.fLocation)
     {
-        //for 2.5 and earlier
-#warning we still store "CreatorLocation" in Defaults.plist, and not "CreatorLocationURL"
```

Comment:
could you keep this `#warning`?

Reply:
I disagree: a warning should be for something needing to be fixed. Here the implementation is precisely as expected.